cpupower_gui/utils.py

```diff
@@ -97,21 +97,22 @@ def read_freqs(cpu):
 
 
 def read_freq_lims(cpu):
```

Comment:
For code symmetry since both `read_freqs` and `read_freqs_lims` read frequency values from `sysfs` it would be nice to add `if is_online`... etc. and format the function the same way as `read_freq_lims`.

That way both will look the same, and by initialising their values to zero here we don't have to do it everywhere else like in `helper.py` line `222`.